test/beast/websocket/read3.cpp

```diff
@@ -16,6 +16,7 @@
 #include <boost/asio/io_context.hpp>
 #include <boost/asio/strand.hpp>
 #include <boost/asio/write.hpp>
+#include <boost/algorithm/hex.hpp>
```

Comment:
This introduces a dependency on Boost.Algorithm. We could probably get rid of that by reformatting the string constant:

```
char const lines[] ={
    0xc1, 0x2d, 0xaa, 0x56, 0xca, ... };
```

Reply:
done